From b5bb80a0ad19bd15f6115861ebb31f41ad79ba00 Mon Sep 17 00:00:00 2001
From: Juergen Gross <jgross@suse.com>
Date: Thu, 29 Sep 2022 13:07:35 +0200
Subject: SUPPORT.md: clarify support of untrusted driver domains with
 oxenstored

Add a support statement for the scope of support regarding different
Xenstore variants. Especially oxenstored does not (yet) have security
support of untrusted driver domains, as those might drive oxenstored
out of memory by creating lots of watch events for the guests they are
servicing.

Add a statement regarding Live Update support of oxenstored.

This is part of XSA-326.

Signed-off-by: Juergen Gross <jgross@suse.com>
Acked-by: George Dunlap <george.dunlap@citrix.com>
Acked-by: Julien Grall <jgrall@amazon.com>
Reviewed-by: Christian Lindig <christian.lindig@citrix.com>

diff --git a/SUPPORT.md b/SUPPORT.md
index 3f4a01101e53..2db341c1d853 100644
--- a/SUPPORT.md
+++ b/SUPPORT.md
@@ -149,6 +149,17 @@ Output of information in machine-parseable JSON format
 
     Status: Supported
 
+## Xenstore
+
+### C xenstored daemon
+
+    Status: Supported
+
+### OCaml xenstored daemon
+
+    Status: Supported
+    Status, untrusted driver domains: Supported, not security supported
+
 ## Toolstack/3rd party
 
 ### libvirt driver for xl
